.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,35 @@
+name: Node CI
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+
+    runs-on: ubuntu-16.04
+
+    strategy:
+      matrix:
+        node-version: [10.x]
```

Comment:
We might be fine to just use the latest version of node as defined in the `setup-node` action and avoid needing a matrix strategy. I think we were only specifying 10.x for Travis because it was required, and the latest version at the time.

Reply:
Thank you for the clarification and pointing out the matrix strategy. I have removed the matrix strategy but still kept the Node to 10.x (the latest stable version). 

I would like to mention one thing, I had to use Ubuntu 16.04 image instead of 18.04 because g++-4.9 does not exist yet on 18.04 apt repository.